perun-core/src/test/java/cz/metacentrum/perun/core/entry/AuditMessagesManagerEntryIntegrationTest.java

```diff
@@ -169,6 +178,75 @@ public void getMessagesPage_ascendingOrder() throws Exception {
 		assertThat(messages.getData().get(0).getId()).isLessThan(messages.getData().get(1).getId());
 	}
 
+	@Test
```

Comment:
Authorization tests are used mainly in `AuthzResolverIntegrationTest`, maybe it would be better to move these to the same file?